zuul.d/jobs.yaml

```
@@ -2139,3 +2139,19 @@
       - OS:ubuntu-xenial
       - Arch:x86_64
       - BuildType:Integration test
+
+# hbase arm build periodic job
+- job:
+    name: hbase-build-arm64
+    parent: init-test
+    description: |
+      Hbase build in openlab arm instance.
+    run: playbooks/hbase-arm-build/run.yaml
+    nodeset: ubuntu-xenial-arm64
+    tags:
+      - Category:BigData
+      - Project:apache/hbase
+      - Application:Hbase@latest-release
+      - OS:ubuntu-xenial
+      - Arch:aarch_64
```

This arch should keep consistent with `uname -i`, and please fix job `kind-integration-test-arm64`' arch option
```suggestion
      - Arch:aarch64
```

---

Thanks, Done